syft/execution/plan.py

```diff
@@ -321,8 +328,12 @@ def build(self, *args):
 
         results = (results,) if not isinstance(results, tuple) else results
 
-        arg_ids = [t.id for t in args if isinstance(t, FrameworkTensor)]
-        result_ids = [t.id for t in results if isinstance(t, FrameworkTensor)]
+        arg_ids = [
+            t.id for t in args if isinstance(t, FrameworkTensor) or isinstance(t, AbstractTensor)
+        ]
+        result_ids = [
```

Comment:
This is needed because we return a ```SyftCrypTensor``` which is an ```AbstractTensor```

Reply:
Why is `SyftCrypTensor` not a `FrameworkTensor`? Is Crypten not a framework?

Reply:
Interesting point. Maybe it is not because currenly SyftCrypTensor is just a helper to build Plans for Crypten, it doesn't serve other purpose than building plans. But if it proves to be more useful, than it should be definitely a FrameworkTensor